src/NServiceBus.Core/OpenTelemetry/Metrics/MeterTags.cs

```diff
@@ -1,9 +1,22 @@
 ﻿namespace NServiceBus;
 
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
 static class MeterTags
 {
     public const string EndpointDiscriminator = "nservicebus.discriminator";
     public const string QueueName = "nservicebus.queue";
     public const string MessageType = "nservicebus.message_type";
     public const string FailureType = "nservicebus.failure_type";
+    public const string MessageHandlerType = "nservicebus.message_handler_type";
+
+    public static TagList BaseTagList(string queueName, string discriminator, string messageType)
```

Comment:
This needs a better name, maybe something like CreateInitializedTagList or something like that to hint that the created tag list already contains something.

Reply:
Since the method provides a set of tags that should always be registered for messaging-related metrics, maybe something like `commonMessaggingMetricTags ` or `mandatoryMessagingMetricTags` could fit?